spec/linter-eslint-spec.js

```diff
@@ -192,12 +192,12 @@ describe('The eslint provider for Linter', () => {
       const editor = await atom.workspace.open(paths.badImport)
       const messages = await lint(editor)
       const expected = "Unable to resolve path to module '../nonexistent'. (import/no-unresolved)"
-      const expectedUrl = 'https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-unresolved.md'
+      const expectedUrlRegEx = /https:\/\/.+\/eslint-plugin-import\/.+\/no-unresolved.md/
```

Comment:
I'm not a fan of using regexp `.+` or `.*` when it's not really necessary, since it can literally match anything. This string could have spaces and new-lines and a bunch of weird symbols in the middle of the "url", and it would all be fine. How about `[/\w-.]+` instead of `.+`? This matches on 1 or more of forward-slash, any alpha-numeric, underscore, dash, and dot.

With that change in place, I think it is reasonably sane if you want to simplify by removing explicit checks for `/`, in which case the full expression would read 
```javascript
/https:[/\w-.]+eslint-plugin-import[/\w-.]+no-unresolved.md/
```
which in my opinion is actually even slightly easier to read than the original pattern, despite the more complex character class description.

Otherwise, everything looks good to me.

Reply:
> This string could have spaces and new-lines and a bunch of weird symbols in the middle of the "url", and it would all be fine.

We aren't testing the validity of the URL they are using, that's their job. We just care that it "looks like" a URL for that rule.

That being said, there isn't a reason _not_ to change that, although I'm going to use `\S` instead of `\w` since URLs are _far_ less restrictive than `\w`.

Reply:
Still a little lose for my taste, but I think `\S` a fair compromise here. I definitely agree we're not really trying to prove its a URL. It's mostly a sanity check on the characters between what we're interested in.